service/src/main/java/com/codeforcommunity/processor/MapProcessorImpl.java

```diff
@@ -182,20 +182,20 @@ public SiteGeoResponse getSiteGeoJson() {
       return SiteGeoResponseCache.getResponse();
     }
 
-    Field<Timestamp> maxDate = max(SITE_ENTRIES.UPDATED_AT).as("maxDate");
+    Field<Timestamp> maxDate = max(SITE_ENTRIES.CREATED_AT).as("maxDate");
 
     Table<
             Record2<
                 Integer, // Site Entry ID
-                Timestamp // Updated At
+                Timestamp // Created At
             >>
-        recentlyUpdated =
+        recentlyCreated =
             table(
```

Comment:
💭 Is this how the code gets auto-formatted? This almost feels harder to read than if it were on one line 🤔

Reply:
Yeah `mvn clean install` doesn't touch this file. We could look into a formatter/linter for the backend too